### ISSUE TITLE
Pairing Process

### ISSUE THREAD

**Issue by jrmurdoch** [devmapal/nvidia-shield-controller-driver#3]
Are you able to give more details on the pairing process? I've tried to monitor the WiFi signals between the Sheild and Controller although I don't see how the pairing takes place. I would have expected to see a AP show up that gets connected to during the pairing process. If I create a WiFi Direct Advertisement as a Group Owner on a PC how do I get the to connect?

**Comment by devmapal** [comment]
Yes, you do see an AP that is created by the Shield tablet. You should be able to see the AP as long as you have the pairing app open and no controller connecting.
In order to pair the controller with a PC, you need to initiate the pairing as an autonomous group owner (see https://en.wikipedia.org/wiki/Wireless_LAN#Peer-to-peer) and enable pairing via WPS push button method. Then you PC will advertise a AP that the shield controller will connect to if you long-press the nvidia button on the controller

**Comment by jrmurdoch** [comment]
I'm working off a Shield TV although I assume the process is the same. With Windows 10, I can create a WiFi Direct Advertiser as an autonomous group owner. Testing with my android phone I can Pair. Unfortunately, I'm unable to get the Controller to connect or pair. I m might try to sniff the packets of the pairing process with the shield TV and see if I can workout whats going on. I'm interested in a building a WiFi Driver for the Controller on Windows. Do you know of any Live Linux distributions or android apps that could be used to test the pairing process?

**Comment by devmapal** [comment]
It should in principle work with any linux distribution. Probably easiest to get started with a Ubuntu live cd, you then just need to install and configure wpa_supplicant on it.

**Comment by Jvs34** [comment]
@jrmurdoch have you had the chance to continue working with the win 10 apis?

I managed to connect my raspberry pi 3 (it has wifi direct support) to the controller with the commands posted in the readme as a test (without the driver ofc, just a mere test).
Tried to use the same win 10 apis you mentioned but no luck here either.
